lib/Digitick/Sepa/TransferFile/Facade/CustomerDirectDebitFacade.php

```diff
@@ -62,7 +62,9 @@ public function addPaymentInfo($paymentName, array $paymentInformation)
             $payment->setLocalInstrumentCode($paymentInformation['localInstrumentCode']);
         }
         $payment->setDueDate($this->createDueDateFromPaymentInformation($paymentInformation, '+5 day'));
-
+        if(isset($paymentInformation['batchBooking'])){
```

Comment:
can you add a space between if and ( thx

Reply:
done @monofone